### PR TITLE
Instructions stored on the main webpage

### PR DESCRIPTION
**Pull request by horcsinbalint**
My recommendations for #450

### CODE REVIEW

**Comment by horcsinbalint** [comment]
TODO: move into blade so data from .env can be used and would also be more elegant

**Review by kdmnk — CHANGES_REQUESTED** [review]
This should still not be part of our codebase. Put a pdf file with the guide on the server and add a link with a configurable path to it.

File: routes/web.php [hunk 1]
```diff
@@ -136,6 +136,8 @@
             'index', 'store', 'update', 'destroy'
         ]);
 
+        Route::get('/help_windows11', [InternetController::class, 'helpWindows11'])->name('help_windows11');
```
Comment (kdmnk) [inline routes/web.php]: delete if not used

**Review by kdmnk — APPROVED** [review]
Just one comment, after that you can merge

**Pull request merged**